cmd/root.go

```diff
@@ -309,24 +313,18 @@ var detectHelmCmd = &cobra.Command{
 	Short: "detect-helm",
 	Long:  `Detect Kubernetes apiVersions in a helm release (in cluster)`,
 	Run: func(cmd *cobra.Command, args []string) {
-		h, err := helm.NewHelm(namespace, kubeContext, apiInstance)
+		exitCode, err := detectHelm()
 		if err != nil {
-			fmt.Printf("error getting helm configuration: %s\n", err.Error())
-			os.Exit(1)
-		}
-		err = h.FindVersions()
-		if err != nil {
-			fmt.Println("Error running helm-detect:", err)
-			os.Exit(1)
+			fmt.Println(err)
+			os.Exit(exitCode)
 		}
 		err = apiInstance.DisplayOutput()
 		if err != nil {
-			fmt.Println("Error Parsing Output:", err)
+			fmt.Printf("Error Parsing Output: %v", err)
```

Comment:
minor: Printf doesn't include a newline by default. There are a few instances of this
```suggestion
			fmt.Printf("Error Parsing Output: %v\n", err)
```

Reply:
Ah thanks - fixed.